app/models/g5_authenticatable/user.rb

```diff
@@ -38,7 +38,7 @@ def update_roles_from_auth(auth_data)
     end
 
     def clients
-      G5Updatable::ClientPolicy::Scope.new(self, G5Updatable::Client).resolve
+      G5Updatable::ClientPolicy::Scope.new(self, G5Updatable::Client).clients_from_client_and_location_roles
```

Comment:
Maybe it's a good idea to have a different Policy.. like `IndirectClientThroughLocationPolicy` or something similar that makes it clear that is not a 100% `pure` role granting, but and indirect or inferred one... what do you think @maeve @jonnysamps ?